package testtask.cellphonevalidator.exceptions;

public record ExceptionResponse(String message) {
}
